TESTING.md

```diff
@@ -140,4 +140,24 @@ mypy .
 
 from the root. See `mypy.ini` for our custom mypy settings.
 
+## Generate documentation from Jupyter notebooks
 
+The documents in the `tutorials` section are automatically generated from the Jupyter notebooks.
+
+The `convert.sh` script in the `scripts/nb2doc/` directory can be used:
```

Comment:
```suggestion
The script `scripts/nb2doc/convert.sh` can be used:
```

Reply:
I originally worded it like this and then changed my mind, but if this reads better I'll revert :)